Unreal Launcher/Unreal Launcher/Project.cs

```diff
@@ -33,7 +33,7 @@ public class Project
 		public ProjectLaunchSettings LaunchSettings = new ProjectLaunchSettings();
 
 		[NonSerialized]
-		public bool projectInitialised = false;
+		public bool ProjectInitialised = false;
```

Comment:
Do not like this. If you must have it publicly accessible then use a property rather than member variable. `public bool IsProjectInitialised { get; private set; } = false;` (or `set;` if it must be publicly settable - though it preferably wouldn't be). And then you can remove the `[NonSerialized]` decorator as properties aren't serialized